.env

```diff
@@ -0,0 +1,4 @@
+PORT=2000
```

Comment:
You should put this file in .gitignore so that it doesn't show up here.

Reply:
i added it because the  port which server runs on is defined in environment variables